src/packages/@ncigdc/modern_components/Counts/Count.js

```diff
@@ -1,12 +1,17 @@
 // @flow
 import React from 'react';
 import { get } from 'lodash';
-import { compose, withProps } from 'recompose';
+import { compose, withProps, withPropsOnChange } from 'recompose';
 
 export default compose(
   withProps(({ viewer, path }) => {
     return {
-      count: get(viewer, path, 0).toLocaleString(),
+      count: get(viewer, path, 0),
     };
   }),
-)(({ count }) => <span>{count}</span>);
+  withPropsOnChange(['count'], ({ count, handleCountChange }) => {
```

Comment:
Could you switch this to use utils/withPropsOnChange, it's the same but won't have an error if setting state during construction.